docs/guides/migrate-to-zeebe-user-tasks.md

```diff
@@ -0,0 +1,376 @@
+        </td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Supports task reports in Optimize</th>
+        <td><NoItem /></td>
+        <td><YesItem /></td>
+    </tr>
+    <tr>
+        <th style={{ textAlign: "end" }}>Recommendations</th>
+        <td>
+            <TableTextSmall>You can continue to use this task type on existing projects when you have a custom task application running on it and do not require any of the above features.</TableTextSmall>
+            <TableTextSmall>Refer to the <a href="#decide-on-your-migration-path">decision helper</a> above for a tailored recommendation.</TableTextSmall>
+        </td>
+        <td>
+            <TableTextSmall>Use this task type on any new projects when you run the OOTB Tasklist.</TableTextSmall>
+            <TableTextSmall>Migrate existing projects and task applications/clients to this task type when you require one of the features above, or the following use cases:</TableTextSmall>
+            <TableTextSmall>
+                <ul>
+                    <li>Implement a full task lifecycle</li>
+                    <li>React on any change/events in tasks, such as assignments, escalations, due date updates, or any custom action</li>
+                    <li>Send notifications</li>
+                    <li>Track task or team performance</li>
+                    <li>Build an audit log on task events</li>
+                    <li>Enrich tasks with business data</li>
+                </ul>
+            </TableTextSmall>
+            <TableTextSmall>Refer to the <a href="#decide-on-your-migration-path">decision helper</a> above for a tailored recommendation.</TableTextSmall>
+        </td>
+    </tr>
+</table>
+
+## Switch the implementation type of your user tasks
+
+We recommend you to migrate process-by-process, allowing you to thoroughly test the processes in your test environments or via your [CI/CD](./devops-lifecycle/integrate-web-modeler-in-ci-cd.md).
+
+1. Open a diagram you want to migrate.
+2. Click a user task.
+3. Check if the task has an embedded form.
+   - In case a form is embedded, [transform it first into a linked form](../components/modeler/bpmn/user-tasks/user-tasks.md#camunda-form-linked), before you change the task type implementation.
+4. Open the **Implementation** section in the **properties panel**.
+5. Click the **Type** dropdown and select **Zeebe user task**. The linked form or external form reference will be preserved.
+
+<img src={ZeebeTaskSelectionImg} className={styles.noShadow} style={{width: 341}} alt="Task Type Selection" />
+
+Repeat these steps for all user tasks in the process. Finally, deploy the process to your dev cluster and test it, e.g. run the process and ensure your custom task applications work.
+
+:::caution
+Make sure to migrate from embedded to linked forms first before changing the implementation type. Use the undo function in case you accidently removed your embedded form.
```

Comment:
By "undo function" do you mean "icon"?

Reply:
No, the key shortcut—let me write that out :)